ext/TenetKrylovKitExt.jl

```diff
@@ -0,0 +1,61 @@
+module TenetKrylovKitExt
+
+using KrylovKit
+using Tenet
+
+"""
+    KrylovKit.eigsolve(tensor::Tensor; left_inds, right_inds, kwargs...)
+
+Perform eigenvalue decomposition on a tensor.
+
+# Keyword arguments
+
+  - `left_inds`: left indices to be used in the eigenvalue decomposition. Defaults to all indices of `t` except `right_inds`.
+  - `right_inds`: right indices to be used in the eigenvalue decomposition. Defaults to all indices of `t` except `left_inds`.
+"""
+function KrylovKit.eigsolve(tensor::Tensor; left_inds=(), right_inds=())
+    return eigsolve(tensor; left_inds=left_inds, right_inds=right_inds)
+end
+
+function KrylovKit.eigsolve(
+    tensor::Tensor,
+    x₀::Vector{ComplexF64},
+    howmany::Int64,
+    which::Union{Symbol,EigSorter},
+    alg::Lanczos;
+    left_inds=(),
+    right_inds=(),
+)
+    return eigsolve(tensor, x₀, howmany, which, alg; left_inds=left_inds, right_inds=right_inds)
+end
+
+function eigsolve(tensor::Tensor, args...; left_inds=(), right_inds=())
```

Comment:
I think you can rewrite it like this:

```suggestion
function KrylovKit.eigsolve(tensor::Tensor, args...; left_inds=(), right_inds=(), kwargs...)
```

Reply:
I tried the same but it did not work, since it conflicts with some other methods of `KrylovKit.eigsolve`.

Reply:
Can you show me the error?

Reply:
I don't have it here, but the it said that there was ambiguity with the `kwcall`.

Reply:
That probably can be solved by adding all the kwargs manually, but we need the exact error.

Reply:
...
This is the error
```julia
  MethodError: kwcall(::@NamedTuple{left_inds::Vector{Symbol}, right_inds::Vector{Symbol}}, ::typeof(eigsolve), ::Tensor{ComplexF64, 2, Matrix{ComplexF64}}, ::Vector{ComplexF64}, ::Int64, ::Symbol, ::Lanczos{ModifiedGramSchmidt2, Float64}) is ambiguous.
  
  Candidates:
    kwcall(::NamedTuple, ::typeof(eigsolve), tensor::Tensor, args...)
      @ TenetKrylovKitExt ~/git/Tenet.jl/ext/TenetKrylovKitExt.jl:32
    kwcall(::NamedTuple, ::typeof(eigsolve), A, x₀, howmany::Int64, which::Union{Symbol, EigSorter}, alg::Lanczos)
      @ KrylovKit ~/.julia/packages/KrylovKit/Y0zG7/src/eigsolve/lanczos.jl:1
  
  Possible fix, define
    kwcall(::NamedTuple, ::typeof(eigsolve), ::Tensor, ::Any, ::Int64, ::Union{Symbol, EigSorter}, ::Lanczos)
```

Reply:
This is difficult to fix in this way, since we would need to define `kwcall` for all the different options available.

Reply:
I almost have it solved in local. Let me fix a couple of things more and it should be done.